Client/Frontend/Sync/SyncAddDeviceViewController.swift

```diff
@@ -114,11 +109,19 @@ class SyncAddDeviceViewController: SyncViewController {
     self.codewordsView.text = syncAPI.getSyncCode()
     self.setupVisuals()
   }
+  
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+    
+    if !syncAPI.isInSyncGroup {
+      DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) { [weak self] in
```

Comment:
Do we need the delay here since we are already showing the error in viewDidAppear ?

Reply:
Fixed.